SH4RE/SH4RE/Components/Calendar/RKColorSettings.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  RKColorSettings.swift
+//  RKCalendar
+//
+//  Copyright © 2019 Raffi Kian. All rights reserved.
+//
+
+import Foundation
+import Combine
+import SwiftUI
+
+class RKColorSettings : ObservableObject {
+
+    // foreground colors
+    @Published var textColor: Color = Color.primary
+    @Published var todayColor: Color = Color.white
+    @Published var selectedColor: Color = Color.primaryDark
+    @Published var disabledColor: Color = Color.gray
+    @Published var betweenStartAndEndColor: Color = Color.white
+    // background colors
+    @Published var textBackColor: Color = Color.white
+    @Published var todayBackColor: Color = Color.gray
+    @Published var selectedBackColor: Color = Color.primaryLight
+    @Published var disabledBackColor: Color = Color.clear
+    @Published var betweenStartAndEndBackColor: Color = Color.blue
+    // headers foreground colors
+    @Published var weekdayHeaderColor: Color = Color.primary
+    @Published var monthHeaderColor: Color = Color.primary
+    // headers background colors
+    @Published var weekdayHeaderBackColor: Color = Color.clear
```

Comment:
Would this be there you change BG colour? Idk i am not terribly worried. I think the UI is not great of this library, but it works and maybe that is okay for now.

Reply:
ya i fiddled w it a bunch but just decided to leave it